src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -105,27 +133,103 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
     setAnchorEl(null);
   };
 
-  const unreviewedActionsDropdown = (
-    <div>
-      <NERButton
-        endIcon={<ArrowDropDownIcon style={{ fontSize: 28 }} />}
-        variant="contained"
-        id="unreviewed-cr-actions-dropdown"
-        onClick={handleClick}
-      >
-        Actions
-      </NERButton>
-      <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
-        <MenuItem onClick={handleReviewOpen} disabled={!isUserAllowedToReview}>
-          Review
-        </MenuItem>
-        <Divider />
-        <MenuItem disabled={!isUserAllowedToDelete} onClick={handleDeleteOpen}>
-          Delete
-        </MenuItem>
-      </Menu>
-    </div>
-  );
+  const handleRequestReviewerClick = async () => {
+    if (reviewerIds.length === 0) return;
+    try {
+      await requestCRReview.mutateAsync({ userIds: reviewerIds });
+    } catch (e) {
+      if (e instanceof Error) {
+        toast.error(e.message);
+      }
+    }
+  };
+
+  const isRequestAllowed = changeRequest.submitter.userId === currentUser.userId;
+
+  const unreviewedActionsDropdown = () => {
+    if (isRequestAllowed && changeRequest.status === ChangeRequestStatus.Open) {
+      return (
+        <div>
+          <NERButton
+            endIcon={<ArrowDropDownIcon style={{ fontSize: 28 }} />}
+            variant="contained"
+            id="unreviewed-cr-actions-dropdown"
+            onClick={handleClick}
+          >
+            Actions
+          </NERButton>
+          <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
+            <MenuItem onClick={handleReviewOpen} disabled={!isUserAllowedToReview}>
+              Review
+            </MenuItem>
+            <Divider />
+            <MenuItem disabled={!isUserAllowedToDelete} onClick={handleDeleteOpen}>
+              Delete
+            </MenuItem>
+          </Menu>
+          <Grid container spacing={2}>
```

Comment:
this should not be happening, the button should always just be one line.
<img width="308" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/99ce1e48-0e8c-4293-8170-537e2eadfbf5">
For now because of how header right, I think the easiest way to achieve this is to just remove all the grid stuff, and have the button be below the autocomplete, like this:
<img width="356" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/453543b7-0fcc-42c5-b825-b4a0578198a1">